src/dataBase/postgres/postgresBookRepo.go

```diff
@@ -17,51 +17,33 @@ func NewBooksRepository(Db *sql.DB) repository.BookRepository {
 
 //GetByCategory iterates over the DB using the SQL SELECT Request and return selected book by its ID
 func (p booksRepositoryPG) GetByID(bookID int) (b repository.Book, err error) {
-	//for connection to HerokuDatabase
-	//db := repository.OpenDb()
-//	db:=dataBase.Connection
-
 	rows, err := p.Db.Query("SELECT title, description, popularity FROM gotoboox.books where id=$1", bookID)
 	if err != nil {
 		log.Fatal(err)
```

Comment:
log.Fatal stops the whole application. I suppose we dont need to stop the server when we cant execute a single query.

Reply:
We have previously divided the tasks. The changes in BookRepo isn't my area of responsibility.